resource-watch/resource-watch
# adds tailwindcss

## Overview

- Adds [tailwindcss](https://tailwindcss.com/) to the project.
- Removes `_deprecated.scss` file.
- Removes `@zeit/next-css` and `@zeit/next-sass`. Points all styles to `css/index.scss` but keeping the style file in the component.
- SASS variable cleanup.
- Updates Storybook to listen to TailwindCSS changes.

## Testing instructions
- run `yarn` to install new dependencies
- test previous styles work as expected.
- adding a Tailwind class should work.

## Jira task / Github issue
–

## Checklist before submitting
